src/cider/nrepl/middleware/info.clj

```diff
@@ -79,7 +79,22 @@
 (defn info-cljs
   [env symbol ns]
   (some-> (cljs-info/info env symbol ns)
-          (select-keys [:file :line :ns :doc :column :name :arglists])))
+          (select-keys [:file :line :ns :doc :column :name :arglists])
+          (update
+            :file
+            (fn [f]
+              (if (boot-project?)
```

Comment:
I can add a unit test for this change, but for the most part it won't be an accurate representation of what boot does. I can emulate a _similar scenario_, but I wasn't sure if this was preferable.